exercises/nucleotide-count/example.el

```diff
@@ -0,0 +1,19 @@
+;;; nucleotide-count.el --- nucleotide-count Exercise (exercism)
+
+;;; Commentary:
+
+;;; Code:
+
+
+(defun nucleotide-count (sequence)
+  (let ((counts (copy-alist '((?A . 0) (?C . 0) (?G . 0) (?T . 0)))))
+    (dolist (i (append sequence nil))
+      (let ((n (assoc i counts)))
+        (if n
+            (setcdr n (1+ (cdr n)))
+          (error "Unknown value '%c' %d" i i))))
```

Comment:
It might be good to replace "Unknown value" with something like "Invalid nucleotide."